client/src/components/Icon/Icon.test.js

```diff
@@ -11,6 +11,11 @@ describe('Icon', () => {
     expect(shallow(<Icon name="test" />)).toMatchSnapshot();
   });
 
+  it('#name for SVG path data', () => {
+    const svgPathData = 'M10 30 L20 10 L30 30 L40 30 L50 10 L60 30';
```

Comment:
```suggestion
    const svgPathData = ['M10 30 L20 10 L30 30 L40 30 L50 10 L60 30'];
```

Additionally,  it would be good for this unit test to do more than just a snapshot. Maybe it should check that there is a `path` element with the appropriate attribute.

Reply:
I will work on this.